### PR TITLE
Hippo mob + Simple mob sounds

### PR DESCRIPTION
**Pull request by lbnesquik**
Adds an hippo with sprites given by kligor.
Adds the possibility for swallow noises.

### CODE REVIEW

**Comment by lbnesquik** [comment]
https://github.com/VOREStation/VOREStation/issues/509

File: code/modules/mob/living/simple_animal/vore/hippo.dm [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+	meat_amount = 10 //Infinite meat!
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
+
+// Activate Noms!
+/mob/living/simple_animal/retaliate/hippo //I don't know why it's in a seperate line but everyone does it so i do it
+	vore_active = 1
+	vore_capacity = 1
+	vore_bump_chance = 15
+	vore_bump_emote = "lazily wraps its tentacles around"
```
Comment (Zeldazackman) [inline code/modules/mob/living/simple_animal/vore/hippo.dm]: Are alien Hippos supposed to have tentacles

Comment (lbnesquik) [reply code/modules/mob/living/simple_animal/vore/hippo.dm]: Totally

**Comment by lbnesquik** [comment]
Active map edge. Totally not shitcode. Plox merge.

**Pull request merged**